lua/cmp/view.lua

```diff
@@ -4,6 +4,7 @@ local event = require('cmp.utils.event')
 local keymap = require('cmp.utils.keymap')
 local docs_view = require('cmp.view.docs_view')
 local custom_entries_view = require('cmp.view.custom_entries_view')
+local statusline_entries_view = require('cmp.view.statusline_entries_view')
```

Comment:
I think the filename and varname shouldbe `wildmenu_entries_view`. (What do you think about it?)

Reply:
I dont really have an opinion here. I dont mind changing if you do.